style.ts

```diff
@@ -549,12 +553,22 @@ export interface ColorMap {
   extended?: boolean;
 }
 
+/**
+ * A VendorOption defines the algorithm to apply for Normalize contrast enhancement.
+ */
+export interface VendorOption {
+  algorithm?: 'StretchToMinimumMaximum' | 'ClipToMinimumMaximum' | 'ClipToZero';
```

Comment:
I see that you are using nullable types for all the properties. If any of this is required I would remove the question mark.

Reply:
For example, it is strange to see a sum type (alghoritm) as nullable, I would add a 'None' type if no alghoritm is possible, and remove the nullable

Reply:
I think the algorithm property alone is required. So I will update the property accordingly